es6 & beyond/ch3.md

```diff
@@ -19,40 +19,41 @@ Embora eles não sejam geralmente usados em JS de tal forma, iteradores também
 
 ### Interfaces
 
-At the time of this writing, ES6 section 25.1.1.2 (https://people.mozilla.org/~jorendorff/es6-draft.html#sec-iterator-interface) details the `Iterator` interface as having the following requirement:
```

Comment:
Pessoal, estaba revisando o texto mais uma vez, antes de mandar, é vi que este link esta quebrado, que fazer nesse caso:
1- Deixar o mesmo?
2- Ou procurar algun novo link para referencia?
 - caso a obção 2 seja escolhina, fiz uma pequena pesquisa no google, é achei o seguinte link explicando mais sobre o asunto, acredito que sejá igual. link abaixo
https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Iteration_protocols


Fico no aguardo dos comentarios para proceguir

Reply:
@osmar-vil existe uma issue aberta com alguns links quebrados desse capitulo.

https://github.com/getify/You-Dont-Know-JS/issues/1249

Na minha opinião, deixamos com o link quebrado mesmo.

Reply:
entendo @guilhermewaess, para manter o conteudo como no original, blz, fica como está então. valew pelo help!